docs/use-linea/explore/use-snapshotx.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Use Snapshot X EVM
+sidebar_position: 1
+---
+
+# Snapshot X
+
+**Snapshot X** is an on-chain voting protocol. Technically speaking, it's a set of modular smart-contracts that interact with each other to do coordinate. The difference with the original [Snapshot](https://snapshot.org) is that Snapshot X is **fully on-chain**.
+
+If anything in these docs is unclear or you would like more detail, do not hesitate to reach out on [Discord](https://discord.gg/snapshot) or view the full documentation at [https://docs.snapshotx.xyz](https://docs.snapshotx.xyz).
+
+Let's quickly go over the core elements of the voting process:
+
+* **Space** - You can think of a space as an organization's account on Snapshot which can be viewed by anyone visiting the platform. It serves as a hub for all proposals related to the organization and a source of information for the users.
+* **Proposal** - Proposal is the key element of the voting system. It presents a change suggestion related to a specific organization and enables eligible users to cast their vote. 
+* **Voting Strategies** - Voting strategies are the contracts used to determine the voting power (VP) of users.
+
+
+# Voting
+
+## Who can vote on proposals? <a href="#who-can-vote-on-proposals" id="who-can-vote-on-proposals"></a>
+
+Each space specifies their [Voting Strategies](https://docs.snapshotx.xyz/protocol-sx-evm/voting-strategies.md) in its settings. You can see the custom setup by opening the space settings. This setup can define if you are eligible to take part in the voting and what is your Voting Power calculated at the timestamp of proposal creation.
```

Comment:
@zzuziak this is a dead link

Reply:
@zzuziak 
can you change, squash your commits, and force push?
lmk if you need help with the process

Reply:
Removed both links 👌  Couldn't squash though, tried to `rebase -i` but your project prompt went through each commit message individually instead 🤔  feel free to fetch and squash!

Reply:
Did you push your changes? the dead link is still there
@zzuziak